mobile/src/reducers/calendar/sick-leave.epics.ts

```diff
@@ -82,4 +82,34 @@ export const sickLeaveCanceledEpic$ = (action$: ActionsObservable<CancelSickLeav
                 { 'Content-Type': 'application/json' }
             ).map(() => loadCalendarEvents(x.employeeId));
         })
+        .catch((e: Error) => Observable.of(loadFailedError(e.message)));
+
+export const sickLeaveApprovedEpic$ = (action$: ActionsObservable<ApproveSickLeave>, state: AppState, deps: DependenciesContainer) =>
+    action$.ofType('APPROVE-SICK-LEAVE')
```

Comment:
Can't we have 'APPROVE-CALENDAR-EVENT' and 'REJECT-CALENDAR-EVENT'?

Reply:
it was my first intention :) but Serj said it's fine to split it by event type, so...

Reply:
so event cancellation is also split by types originally

Reply:
But eventually if it is looked massive, that something wrong